src/variables/Variable.php

```diff
@@ -34,6 +34,46 @@ public function lists(): mixed
         return $this->lists;
     }
 
+    /**
+     * profileFromList.
+     *
+     * @author	Unknown
+     * @since	v0.0.1
+     * @version	v1.0.0	Monday, November 14th, 2022.
+     * @access	public
+     * @param	mixed	$listId	
+     * @param	mixed	$email 	
+     * @return	object
+     */
+    public function profileFromList($listId, $email)
```

Comment:
Need to include Typing on this method. Indicate what type of thing the params are, and the type of thing that is being returned.